FASMA/interpolation.py

```diff
@@ -262,22 +261,20 @@ def interpolator(
     params = list(params)
     if atmtype == 'marcs':
         newatm = interpolator_marcs(params, fesun=7.47, microlim=3.0)
-    elif atmtype == 'kurucz95':
-        newatm = interpolator_kurucz(params, atmtype=atmtype)
     elif atmtype == 'apogee_kurucz':
         newatm = interpolator_kurucz(params, atmtype=atmtype)
     else:
-        raise NameError(f'Could not find {atmtype} models')
+        raise NameError('Could not find %s models', atmtype)
```

Comment:
Why this change? Not a fan of f-strings?

Reply:
The problem is that f-string require python=>3.6. They don't work for 3.5.

Reply:
Oh. That's a shame. It's so elegant.

Reply:
I agree

Reply:
Do you think we can merge this branch now? Then we can move on with the tests

Reply:
Yes. let's merge